components/movementsensor/gpsrtkserial/gpsrtkserial.go

```diff
@@ -637,3 +657,77 @@ func (g *rtkSerial) Close(ctx context.Context) error {
 	}
 	return nil
 }
+
+// sendGGAMessage sends GGA messages to the serial port. This is only used to get NTRIP stream
+// from a virtual reference point.
+func (g *rtkSerial) sendGGAMessage() {
+	// Open the serial port for reading
+	serialPort, err := slib.Open(slib.OpenOptions{
+		PortName:        g.writePath,
+		BaudRate:        uint(g.wbaud),
+		DataBits:        8,
+		StopBits:        1,
+		MinimumReadSize: 1,
+	})
```

Comment:
If these are commonly shared with any other part of the code in this package, let's set them as constants at the top of the file. It may be worth it to consider this for an eventual refactor, since I believe some of these rates and serialibrayr options rely on the underlying `nmeaMovementSensor` implementation, which we shouldn't expose in this driver long term.

My vision for this is just to have the nmea data parser be the package with ANY exposed export functions. And we'd rely on each GPS driver to read an write to a serial port. But too much work for now.